zubhub_backend/zubhub/notifications/tests/test_views.py

```diff
@@ -0,0 +1,126 @@
+from django.urls import reverse, reverse_lazy
+from rest_framework.test import APIClient
+from django.test import TestCase
+from notifications.models import Notification
+from creators.models import Creator
+from notifications.serializers import NotificationSerializer
+from rest_framework import status
+from django.utils import timezone
+import json
+from django.test import Client
+from rest_framework.authtoken.models import Token
+from rest_framework import status
+from django.contrib.auth import get_user_model
+
+creator = get_user_model()
+
+class UserNotificationsAPIViewTest(TestCase):
+
+    @classmethod
+    def setUpTestData(self):
+        self.first_creator = Creator.objects.create(username="krishna")
+        self.second_creator = Creator.objects.create(username="kali")
+        self.client = APIClient()
+
+        self.payload = Notification.objects.create(
+            link="https://zubhub.com/exampleclap",
+            type=Notification.Type.CLAP,
+            recipient=self.first_creator,
+            message="Clap",
+            viewed=False
+        )
+        self.payload.save()
+        self.payload.sources.add(self.second_creator, self.second_creator)
+        self.payload.save()
+
+        self.auth = {
+            "username": "test_dummy2",
+            "email": "test_dummy2@gmail.com",
+            "password1": "qwertyu12345",
```

Comment:
Do not add username and passwords to the codebase. Get the username and passwords through environment variables

Reply:
@chaitanya-cloudknox I am not adding credentials in the main project codebase but only in the test case files. Referring to the current test cases here: https://github.com/unstructuredstudio/zubhub/blob/master/zubhub_backend/zubhub/creators/tests/test_views.py